spec/services/discovery_report_spec.rb

```diff
@@ -102,13 +102,12 @@
     end
 
     it 'process_dobj gives expected output for one dobj' do
-      allow(dobj).to receive(:druid).and_return(DruidTools::Druid.new('kk203bw3276'))
       expect(report.send(:process_dobj, dobj).as_json).to eq(
-        druid: 'druid:kk203bw3276',
-        errors: { dupes: true },
+        druid: 'druid:jy812bp9403',
+        errors: {},
         counts: {
-          total_size: 254_802,
-          mimetypes: { 'image/tiff' => 4, 'image/jp2' => 2 },
+          total_size: 127_401,
+          mimetypes: { 'image/tiff' => 2, 'image/jp2' => 1 },
           filename_no_extension: 0
         }
```

Comment:
1. There is no need to mock the druid, let's use the actual druid from the fixture folder.  This makes it less confusing to see what test data we are using.
2. The previously removed class was actually doing the wrong thing when items had a hierarchy and was counting them twice.  If you look at the fixture data being used here (https://github.com/sul-dlss/pre-assembly/tree/main/spec/fixtures/images_jp2_tif/jy812bp9403) you will see it has a folder hierarchy, with a total of 2 tiffs and 1 jp2.  We were incorrectly double counting before and incorrectly reporting dupes.